docs/tutorials/google/floquet_calibration_example.ipynb

```diff
@@ -212,7 +212,18 @@
     "id": "a0Nrd1pkVWzo"
```

Comment:
<p>single electron =&gt; single fermionic particle</p><p> </p><p>quirk circuit is cool! Could you place the excitation in the middle (on the 2nd qubit, this is how it's being done in the tutorial).</p><p> </p><p>Otherwise LGTM!</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/quantumlib/Cirq/pull/4335/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>

Reply:
<p>Sounds good, I'll add those changes in! </p>